README.md

```diff
@@ -47,6 +47,13 @@ For an authentication ceremony use the following two methods:
 - `WebAuthnManager.beginAuthentication()`
 - `WebAuthnManager.finishAuthentication()`
 
+## Contributing
+
+If you add any new files, please run the following command at the root of the repo to identify any missing license headers:
+```bash
+% /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/swiftlang/github-workflows/refs/heads/main/.github/workflows/scripts/check-license-header.sh)"
```

Comment:
This is subject to the following PR getting merged, otherwise a `PROJECTNAME="Swift WebAuthn" ` is needed:
- https://github.com/swiftlang/github-workflows/pull/87

Reply:
Actually, leaving this in for now and I'll remove this bit along with the option in the workflow if and when that PR gets approved.

Reply:
See #105